lib/src/commands.dart

```diff
@@ -170,6 +170,9 @@ class BuildCommand extends Command {
     for (var packageName in packages.keys) {
       var destPackageDir = _path([destination, 'packages', packageName]);
       var sourcePath = packages[packageName].path;
+      if (Platform.isLinux) {
+        sourcePath = "$sourcePath.";
+      }
```

Comment:
I just tested this on osx and it seems to work with added `.` too. Maybe we can get rid of the platform check?

(The whole command would definitely not work on Windows, but this is not a priority atm, I think.)